interactions/four_particle/__init__.py

```diff
@@ -144,10 +144,14 @@ def integrate(self, ps, stepsize=None):
         if not self.cMs:
             self.cMs = [M_t(list(M.order), M.K1 / unit, M.K2 / unit) for M in self.Ms]
 
-        if environment.get('SPLIT_COLLISION_INTEGRAL'):
-            A = integration(ps, *bounds, self.creaction, self.cMs, stepsize, CollisionIntegralKind.F_1)
-            B = integration(ps, *bounds, self.creaction, self.cMs, stepsize, CollisionIntegralKind.F_f)
+        if environment.get('SPLIT_COLLISION_INTEGRAL') and self.kind in [0, 3]:
+            A = integration(ps, *bounds, self.creaction, self.cMs, stepsize, self.kind + 1)
+            B = integration(ps, *bounds, self.creaction, self.cMs, stepsize, self.kind + 2)
```

Comment:
You destroyed the whole idea of having constants. You shouldn't do algebra with these. Values are unimportant, they are arbitrary — only the variable names are important